Neos.Eel/Classes/Helper/DateHelper.php

```diff
@@ -57,6 +71,31 @@ public function format($date, $format)
     }
 
     /**
+     * Format a date to a string with a given cldr format
+     *
+     * @param \DateTime $date
+     * @param string $cldrFormat Format string in CLDR format (see http://cldr.unicode.org/translation/date-time)
+     * @param string $locale String locale - example (de|en|ru_RU). Must be one of Neos\Flow\I18n\Cldr\Reader\DatesReader::FORMAT_LENGTH_*'s constants.
+     *
+     * @return string
+     */
+    public function formatCldr($date = null, $cldrFormat = null, $locale = null)
+    {
+        if ($date === null) {
+            $date = new \DateTime('now');
+        }
+        if ($cldrFormat === null) {
+            $cldrFormat = 'dd MMMM yyyy HH:mm:ss';
+        }
+        if ($locale === null) {
```

Comment:
Good point, bu then please be explicitly unstrict ;) i.e. `$cldrFormat === null || $cldrFormat === ''`
Relying on the PHP type juggling behaviour is a fallacy